kube-core/src/response.rs

```diff
@@ -1,20 +1,20 @@
 //! Generic api response types
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 
 /// A Kubernetes status object
 ///
 /// Equivalent to Status in k8s-openapi except we have have simplified options
```

Comment:
I would remove this line while we are here, there are more ergonomic differences now

Reply:
Done